core/server/src/main/java/alluxio/worker/block/BlockWorkerClientServiceHandler.java

```diff
@@ -187,6 +188,33 @@ public Void call() throws AlluxioException, IOException {
   }
 
   /**
+   * Convert a relative policy such as HIGHEST or LOWEST into a tier id.
+   *
+   * @param tTierPolicy the relative policy used for block allocation
+   * @return the tier level to use for allocation
+   */
+  private int getLevelFromPolicy(final TTierPolicy tTierPolicy) {
+    if (tTierPolicy == null) {
+      return 0;
```

Comment:
Here's where we preserve backward compatibility with older clients. I tested with a 1.3 client against a server with this patch.